tests/functional/test_libraries.py

```diff
@@ -70,10 +67,6 @@ def test_PyQt4_QtWebKit(pyi_builder):
 @pytest.mark.xfail(reason='Reports "ImportError: No module named QtWebKit.QWebView".')
 @importorskip('PyQt4')
 def test_PyQt4_uic(tmpdir, pyi_builder, data_dir):
-    # Copy the data/PyQt4-uic.ui file to the tempdir used by this test.
-    os.mkdir(os.path.join(tmpdir.strpath, 'data'))
-    shutil.copy(os.path.join(data_dir, 'PyQt4-uic.ui'),
-                os.path.join(tmpdir.strpath, 'data'))
-
+    # Note that including the data_dir fixture copies files needed by this test.
```

Comment:
For me stupid: Can you please explain where this fixture is included. I can not spot it. (I have to admit that I did not yet understand many of the concepts of py.test.)